cltk/utils/matrix_corpus_fun.py

```diff
@@ -21,24 +22,24 @@
 LOG.addHandler(logging.NullHandler())
 
 
-def drop_all_caps(X):
+def drop_all_caps(string_matrix: List[List[str]]) -> List[List[str]]:
```

Comment:
This is fun, I love all these functions.

Reply:
Yes, I'll posting a few notebooks soon, and pushing a CLTK tutorial which will show how they can be used.